Sources/XcodeGenCLI/CommandRouter.swift

```diff
@@ -1,20 +0,0 @@
-import Foundation
-import SwiftCLI
-
-class CommandRouter: Router {
```

Comment:
`Router` seems to be removed from latest SwiftCLI. Using `ArgumentListManipulator` instead.